### ISSUE TITLE
Student progress is not showing under "progress" on page Auview

### ISSUE THREAD

**Issue by ADLMeganBohland** [adlnet/cmi5-player-moodle-plugin#9]
The plugin polls the LRS and displays xAPI statements in a user friendly way. However, these statements are not currently showing up as they should. The area is blank.

**Comment by ADLMeganBohland** [comment]
This is how the page should look:
![image](https://github.com/adlnet/cmi5-player-moodle-plugin/assets/99821412/36a454cc-289c-44e9-b118-60c17e44fdcd)

And currently it is displaying:
![image](https://github.com/adlnet/cmi5-player-moodle-plugin/assets/99821412/c69dbdbf-d47d-4190-8b71-da96dfbcd3ac)

The idea is that it is supposed to query the LRS and display returned statements in an easy to read way for the user. This might be able to be used later with the logstore.

**Comment by ADLMeganBohland** [comment]
Turns out this was not an issue with our plugin at all! Learning Locker has different data 'stores'. The client being used was pointing toward the wrong store. By adjusting Learning Locker our queries worked again. In the future this can be solved in two ways. One - make sure the plugin is using the same LRS client as the player. OR two, if the plugin has a separate LRS client than the player, make sure both clients are accessing the same store. This issue is specific to Learning Locker, not all LRS's, so I don't think it needs to be reflected in our readme? 

I did verify this fix both on my VM and Shweta checked her version, both are displaying correctly now.
